### PR TITLE
Add esp_v2 gke deployment files

### PR DESCRIPTION
**Pull request by qiwzhang**
Signed-off-by: Wayne Zhang <qiwzhang@google.com>

Add two GKE deployment files for ESPv2.

### CODE REVIEW

**Comment by TAOXUY** [comment]
Could you be specific about how they are used or referenced?
I see there a README.md under k8s/. Updated it also?

**Comment by qiwzhang** [comment]
They are used in public doc:

endpoints/docs/_shared/_enabling-ssl.md
endpoints/docs/openapi/get-started-kubernetes-engine.md

File: k8s/esp_v2_echo_gke.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+        args: [
+          "--listener_port", "8080",
+          "--backend", "127.0.0.1:8081",
+          "--service", "SERVICE_NAME",
+          "--rollout_strategy", "managed",
+        ]
+        ports:
+          - containerPort: 8080
+      - name: echo
+        image: gcr.io/endpoints-release/echo:latest
```
Comment (nareddyt) [inline k8s/esp_v2_echo_gke.yaml:53]: So is this for a specific language? Or will we use this yaml file for all languages?

Comment (qiwzhang) [reply k8s/esp_v2_echo_gke.yaml:53]: No,  gke yaml files added here are not language specific.

**Comment by qiwzhang** [comment]
Hold on,  I will update README too

**Pull request merged**